src/Extensions/DataGrid.EntityFrameworkAdapter/Microsoft.FluentUI.AspNetCore.Components.DataGrid.EntityFrameworkAdapter.csproj

```diff
@@ -1,43 +1,42 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-	<PropertyGroup>
-		<TargetFrameworks>net8.0;net9.0</TargetFrameworks>
-		<PackageId>Microsoft.FluentUI.AspNetCore.Components.DataGrid.EntityFrameworkAdapter</PackageId>
-		<Authors>Microsoft</Authors>
-		<Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-		<NeutralLanguage>en</NeutralLanguage>
-		<Owners>Microsoft Corporation</Owners>
-		<PackageProjectUrl>https://github.com/microsoft/fluentui-blazor</PackageProjectUrl>
-		<PackageLicenseExpression>MIT</PackageLicenseExpression>
-		<Summary>An adapter for using the Fluent UI Blazor DataGrid with Entity Framework.</Summary>
-		<PackageTags>Fluent UI, Blazor, DataGrid, .NET8</PackageTags>
-		<Title>Microsoft Fluent UI DataGrid adapter for Entity Framework</Title>
-		<Description>An adapter for using the Fluent UI Blazor DataGrid with Entity Framework.</Description>
-		<SignAssembly>false</SignAssembly>
-		<Nullable>enable</Nullable>
-		<ImplicitUsings>enable</ImplicitUsings>
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net9.0;net10.0</TargetFrameworks>
```

Comment:
The current GitHub Action doesn't seem to support the .NET 10 :-(
```
The current .NET SDK does not support targeting .NET 10.0. 
```

Reply:
I still need to add the SDK to the workflow. Will be in next commit